src/screens/login/components/login-form/login-form.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useCallback } from 'react';
+import { Alert, View } from 'react-native';
+import { Formik } from 'formik';
+
+import { styles } from '../../styles';
+import { MainButton } from 'src/components';
+import { defaultLoginPayload } from '../../common';
+import { authenticateUserValidationSchema } from 'src/validation-schemas';
+import { FormInput, FormPasswordInput } from 'src/components';
+import { IAuthenticateUser } from 'src/common/types';
+import { useAppDispatch } from 'src/hooks';
+import { setCurrentUser, signIn } from 'src/store/auth/actions';
+
+const LoginForm: React.FC = () => {
+  const dispatch = useAppDispatch();
+
+  const handleLogin = useCallback(
+    loginPayload => dispatch(signIn(loginPayload)),
+    [dispatch]
+  );
+
+  const handleAuthorizedUserData = useCallback(
+    () => dispatch(setCurrentUser()),
+    [dispatch]
+  );
+
+  const handleLoginPressed = (values: IAuthenticateUser) => {
+    handleLogin(values)
+      .unwrap()
+      .catch((err: Error) => {
+        // Use react alers/toaster notifications
+        Alert.alert(`${err.message}`);
+      });
+    handleAuthorizedUserData()
+      .unwrap()
+      .catch((err: Error) => {
+        Alert.alert(`${err.message}`);
+      });
+  };
```

Comment:
We don't need useCallback here
```suggestion
  const handleLogin = (values: IAuthenticateUser) => {
    dispatch(authActions.signIn(values))
      .unwrap()
      .catch((err: Error) => {
        // Use react alers/toaster notifications
        Alert.alert(`${err.message}`);
      });
  };
```